main/main.ts

```diff
@@ -140,6 +140,15 @@ app.on('open-url', (e: Event, u: string) => {
     e.preventDefault();
     shell.openExternal(u);
 });
+app.once('will-finish-launching', function() {
```

Comment:
It will certainly work, but the doc says put it inside `will-finish-launching`. So I did as it says.
![2016-10-15 11 51 43](https://cloud.githubusercontent.com/assets/4080524/19407221/c564f060-92cd-11e6-8821-92a7f1695a32.jpg)